Week1task10/src/StringObject.java

```diff
@@ -0,0 +1,42 @@
+
+public class StringObject {
+	
+	String one;
+	String two;
+
+	/**
+	 * @param one
+	 * @param two
+	 */
+	public StringObject(String one, String two) {
+		super();
+		this.one = one;
+		this.two = two;
+	}
+
+	public StringObject() {
+		// TODO Auto-generated constructor stub
+	}
+
+	public static void main(String[] args) {
+		StringObject name = new StringObject();
+		StringObject surname = new StringObject();
+		name.one = "Justyna";
+		surname.two = "Garyga";
+		StringObject connected = new StringObject();
+		// I don't know, what's next
+		// methods name.concat(surname) and length(connected) don't work
+		
+		// solution of task without Objects
+		String stringOne= "abcdefghij";
```

Comment:
This is what I wanted in fact. This is good. The code from lines 31-39 is what the exercise meant. So it's perfect. stringOne is a String object. I didn't want the creation of another class. Just this create 2 Strings and instantiate them. And you did this here. So you can delete code from lines 4-19 and 22-28.